src/service/sandbox-client-manager.ts

```diff
@@ -199,14 +194,8 @@ export async function executeSandboxClient(printLogs) {
     `"${await envUtils.getJavaExecutablePath()}"`
   ];
 
-  if (versionToNumber(CONNECTOR_VERSION) >= versionToNumber('1.3.1')) {
```

Comment:
@krzyk I believe we can get rid of it or I'm wrong? :)

Reply:
yes